recipes/abseil/all/conanfile.py

```diff
@@ -35,9 +35,19 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def configure(self):
-        if self.settings.compiler.cppstd:
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
             tools.check_min_cppstd(self, 11)
+        version = tools.Version(self.settings.compiler.version)
+        compiler = self.settings.compiler
+        if compiler == "gcc" and version < "5.1":
+            raise ConanInvalidConfiguration("abseil requires gcc 5.1+")
+        if compiler == "clang" and version < "3.7":
+            raise ConanInvalidConfiguration("abseil requires clang 3.7+")
+        if compiler == "apple-clang" and version < "7.3":
+            raise ConanInvalidConfiguration("abseil requires Xcode 7.3.1+")
+        if compiler == "Visual Studio" and version < "14":
+            raise ConanInvalidConfiguration("abseil requires at least Visual Studio 2015 Update 3")
```

Comment:
why being so strict. We know it compiles with gcc 4.9 and clang 3.4

Reply:
Here is example user code builds but doesn't links with gcc 4.9. Maybe you can see if there is a problem with the user code, I don't have gcc 4.9 to check.
https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/6330/7/bloaty/1.1//summary.json


BUILT + TEST_OK | bloaty/1.1 | c9aa50b64a4a418b96f07ff2342874cc18e9dc67 | Linux | Linux | x86_64 | x86_64 | gcc | libstdc++ | 4.9 |   | Release | profile | build | test
-- | - | - | - | - | - | - | - | - | - | - | - | - | - | -
FAILED | bloaty/1.1 | 5e02e2869869a374987d358efbe9f1126c0a502b | Linux | Linux | x86_64 | x86_64 | gcc | libstdc++ | 4.9 |   | Debug | profile | build

Reply:
@SpaceIm I think I have figured out the main issue related to the CPP_STANDARD mismatch between upstream and downstream. So, the validation is irrelevant. Please let me know if you want me to remove the adding checks.

Reply:
now you can remove these checks I think